src/lib/Components/App.php

```diff
@@ -88,7 +94,18 @@ public function jsonSerialize()
                     $this->toolbars,
                     [$this->navigationHub, $this->mainContent]
                 ),
-            ],
+            ];
+        }
+
+        return [
+            'selector' => self::TAG_NAME,
+            'update' => $update,
         ];
     }
+
+    private function isError()
+    {
+        return count($this->notifications) === 1
```

Comment:
that does not seem super reliable :) Maybe you can set a property on the app in `RequestAppResponseRenderer::configureAppException`